packages/content/src/requests/items/shareAttribute/ShareAttributeAcceptResponseItem.ts

```diff
@@ -17,7 +17,7 @@ export class ShareAttributeAcceptResponseItem extends AcceptResponseItem impleme
     @validate()
     public attributeId: CoreId;
 
-    public static override from(value: IShareAttributeAcceptResponseItem | ShareAttributeAcceptResponseItemJSON): ShareAttributeAcceptResponseItem {
+    public static override from(value: IShareAttributeAcceptResponseItem | Omit<ShareAttributeAcceptResponseItemJSON, "@type">): ShareAttributeAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here